src/service/iot_config.proto

```diff
@@ -40,27 +40,26 @@ enum action_v1 {
 // Define an organisation
 message org_v1 {
   uint64 oui = 1;
-
   // Org admin key
   bytes owner = 2;
   // Key only used for DC payments
   bytes payer = 3;
   // List of keys allowed some specific actions, see services.
   repeated bytes delegate_keys = 4;
-  // Use req timestamps to validate order/avoid replay
-  uint64 nonce = 5 [ deprecated = true ];
 }
 
 // Device address range, ex: 16#00000001 to 16#0000000A
 message devaddr_range_v1 {
-  uint32 start_addr = 1;
-  uint32 end_addr = 2;
+  string route_id = 1;
+  uint32 start_addr = 2;
+  uint32 end_addr = 3;
 }
 
 // Device App EUI and Dev EUI
-message eui_v1 {
-  uint64 app_eui = 1;
-  uint64 dev_eui = 2;
+message euis_v1 {
```

Comment:
```suggestion
message eui_v1 {
```
i like keeping this singular so you don't see it referenced in the context of the code and mistake it for a message/field comprising multiple eui pairs. could rename `eui_pair_v1` if you want to reflect the fact it contains two eui fields

Reply:
sounds good to me